<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Configuración</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Estadísticas</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bienvenido </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ventanaconfiguracion()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>289</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>ventanaestadisticas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ventanaconfiguracion()</slot>
  <slot>ventanaestadisticas()</slot>
 </slots>
</ui>
